java/src/jmri/jmrix/lenz/XNetTrafficController.java

```diff
@@ -140,45 +138,17 @@ public void forwardReply(AbstractMRListener client, AbstractMRReply m) {
         }
     }
 
-    // We use the pollMessage routines for high priority messages.
-    // This means responses to time critical messages (turnout off messages).
-    // PENDING: these fields should be probably made private w/ accessor to force proper synchronization for reading.
-    final LinkedBlockingQueue<XNetMessage> highPriorityQueue;
-    final LinkedBlockingQueue<XNetListener> highPriorityListeners;
-
-    public synchronized void sendHighPriorityXNetMessage(XNetMessage m, XNetListener reply) {
-        // using offer as the queue is unbounded and should never block on write.
-        // Note: the message should be inserted LAST, as the message is tested/acquired first
-        // by the reader; serves a a guard for next item processing.
-        highPriorityListeners.add(reply);
-        highPriorityQueue.add(m);
+    public void sendHighPriorityXNetMessage(XNetMessage m, XNetListener reply) {
+        super.sendMessage(m.asPriority(true), reply);
     }
-
+    
     @Override
     protected AbstractMRMessage pollMessage() {
-        try {
-            if (highPriorityQueue.peek() == null) {
-                return null;
-            } else {
-                return highPriorityQueue.take();
-            }
-        } catch (java.lang.InterruptedException ie) {
-            log.error("Interrupted while removing High Priority Message from Queue");
-        }
         return null;
     }
-
+    
     @Override
     protected AbstractMRListener pollReplyHandler() {
-        try {
-            if (highPriorityListeners.peek() == null) {
-                return null;
-            } else {
-                return highPriorityListeners.take();
-            }
-        } catch (java.lang.InterruptedException ie) {
-            log.error("Interrupted while removing High Priority Message Listener from Queue");
-        }
```

Comment:
If pollMessage and pollReplyHandler are now returning null, check and see if they can be removed.  I think there is a default implementation in AbstractMRTrafficController

Reply:
OTOH, there are abstracts. Can be changed to no-op methods.